pkg/shell/manifest.json.in

```diff
@@ -24,7 +24,7 @@
     },
     "bridges": [
         {
-            "privileged": true,
+            "privileged": false,
```

Comment:
Does this still have any effect now? Preferring sudo is fine of course (I agree), but in cases where sudo doesn't work, pkexec should still be available as a fallback. We did get bug reports about how to configure polkit for allowing a user to run cockpit-bridge as root. (This of course isn't a real subset of root privileges, but it's a nice picket fence to avoid accidents)

Reply:
> Does this still have any effect now?

Setting `privileged: false` would essentially comment out this bridge. This doesn't work anymore with the extra "privileged XOR match" validation.